### ISSUE TITLE
ci/e2e: test deployment of multiple node in parallel

### ISSUE THREAD

**Issue by ldevulder** [rancher/elemental#442]
There are 2 differents things here:
- OS provisioning in parallel (this should stress `elemental-operator` a bit)
- cluster deployment (K3s and RKE2) on multiple node in parallel

**Comment by ldevulder** [comment]
In fact this issue is a duplicate of #423.
